src/vs/workbench/browser/parts/media/compositepart.css

```diff
@@ -14,4 +14,6 @@
 .monaco-workbench .part > .composite.title > .title-actions {
 	flex: 1;
 	padding-left: 5px;
-}
\ No newline at end of file
+	overflow: hidden;
```

Comment:
@sbatten thoughts on this?

Reply:
my main concern would be the element this applies to having a dropdown or submenu. if it does this may cause issues with drawing that submenu/dropdown. if you tested the various actions this affects on windows and you don't see any clipping, then its probably safe

Reply:
The only other place this applies looks fine to me with this change.

Reply:
Let's check the terminal/output dropdown on Mac (native, should be fine) and windows (custom).

Reply:
Mac looks fine

Reply:
This appears to impact the debug dropdown: https://github.com/microsoft/vscode/issues/128368